core/include/mm/tee_mmu_types.h

```diff
@@ -60,7 +60,15 @@
 #define TEE_MATTR_CACHE_CACHED	1
 
 #define TEE_MATTR_LOCKED		BIT(15)
+/*
+ * Tags TA mappings which are only used during a single call (open session
+ * or invoke parameters).
+ */
 #define TEE_MATTR_EPHEMERAL		BIT(16)
+/*
+ * Tags TA mappings that must not be removed (kernel mappings while in user
+ * mode).
+ */
 #define TEE_MATTR_PERMANENT		BIT(17)
```

Comment:
How about introducing `UTA` (user mode TA) in the names? Sounds like important information, no?
I.e., `TEE_MATTR_UTA_EPHEMERAL` and `TEE_MATTR_UTA_PERMANENT`.

Reply:
No, I'd rather not.

Reply:
OK ;-)